.github/workflows/build-test.yaml

```diff
@@ -19,37 +21,58 @@ env:
 jobs:
 
   build:
+    name: "${{ matrix.tests-type }} tests: py${{ matrix.python-version }} on ${{ matrix.os }} (${{ matrix.test-runner }})"
     strategy:
       matrix:
         os: [
           macos-latest,
           windows-latest,
           ubuntu-latest,
         ]
-        python-version: [3.9]
+        python-version: ['3.10']
         dependencies: [full]
         tests-type: [unit]
+        test-runner: [pytest]
         include:
           - os: ubuntu-latest
-            python-version: 3.7
+            python-version: '3.7'
```

Comment:
Will we ever need to update this?

Reply:
yup, when we drop support for Python 3.7